ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -566,7 +566,8 @@ private class JdbcLedgerDao(
               val submitterInfo =
                 for (
                   appId <- tx.applicationId;
-                  actAs <- if (tx.actAs.isEmpty) None else Some(tx.actAs); cmdId <- tx.commandId
+                  actAs <- if (tx.actAs.isEmpty) None else Some(tx.actAs);
```

Comment:
Probably don't need the semicolons here.

Reply:
semicolons are needed because the enclosing for-comprehension uses parenthesis and not braces.

Reply:
I suggest we stop that, but it's up to you.